Tasks/ANT/task.json

```diff
@@ -191,6 +191,7 @@
             "visibleRule": "javaHomeSelection = JDKVersion",
```

Comment:
you will need to bump the version numbers for all the tasks as well (unless it was already done this sprint)

Reply:
Done.